packages/serverpod/lib/src/database/database_result.dart

```diff
@@ -7,9 +7,13 @@ Map<String, dynamic>? resolvePrefixedQueryRow(
   Map<String, Map<String, dynamic>> rawRow,
   Map<String, Map<int, List<dynamic>>> resolvedListRelations, {
   Include? include,
+  bool viewTable = false,
```

Comment:
Question 1: How does this work with the recursive call on line 47? It looks like we need to assert that include and view table are never used together right?

Question 2: If we don't want the recursive calls, would it make sense to have a specific inplementation for `resolvedPrefixedQueryRow` that only operates on viewTables and shares code with this method.

Question 3: Could we encode this information directly in the table instead of having to pass it in as separate variable? Maybe a specific type table type could be used for view tables?

Reply:
I think there is no need to process the relation table for viewTable since it is already defined in the raw query inside the protocol. I will think to have specific implementation for this method.